config/cypress.json

```diff
@@ -10,7 +10,7 @@
   "testFiles": "**/tests/**/*.cypress.spec.js?(x)",
   "waitForAnimations": false,
   "retries": {
-    "runMode": 2,
+    "runMode": 0,
```

Comment:
@stephenBarrs mind flipping this back to "2"? Unless you are wanting to make this change platform wide...?

Reply:
Whoops!  changed it back.